infra/build_specified_commit.py

```diff
@@ -22,88 +22,102 @@
 import subprocess
 
 import helper
-import repo_manager
 
 
-class DockerExecutionError(Exception):
-  """An error that occurs when running a docker command."""
+class BuildData():
+  """List of data requried for bisection of errors in OSS-Fuzz projects.
 
+  Attributes:
+    project_name: The name of the OSS-Fuzz project that is being checked
+    engine: The fuzzing engine to be used
+    sanitizer: The sanitizer to be used
+    architecture: CPU architecture to build the fuzzer for
+  """
+  # pylint: disable=too-few-public-methods
+  project_name = ''
+  engine = ''
+  sanitizer = ''
+  architecture = ''
 
-def build_fuzzer_from_commit(project_name,
-                             commit,
-                             local_store_path,
-                             engine='libfuzzer',
-                             sanitizer='address',
-                             architecture='x86_64',
-                             old_repo_manager=None):
+
+def build_fuzzers_from_commit(commit, build_repo_manager, build_data):
   """Builds a OSS-Fuzz fuzzer at a  specific commit SHA.
 
   Args:
-    project_name: The OSS-Fuzz project name
-    commit: The commit SHA to build the fuzzers at
-    local_store_path: The full file path of a place where a temp git repo is stored
-    engine: The fuzzing engine to be used
-    sanitizer: The fuzzing sanitizer to be used
-    architecture: The system architiecture to be used for fuzzing
-
+    commit: The commit SHA to build the fuzzers at.
+    build_repo_manager: The OSS-Fuzz project's repo manager to be built at.
+    build_data: A struct containing project build information
   Returns:
     0 on successful build 1 on failure
```

Comment:
I don't think it's necessary 1 on failure, as you seem to return the error code returned by the `docker_run` command, i.e. can be anything, but must be 0 on success.

Reply:
Fixed.